src/sdcv.cpp

```diff
@@ -201,7 +202,10 @@ try {
         if (ordering_file != nullptr) {
             std::string line;
             while (stdio_getline(ordering_file, line)) {
-                order_list.push_back(bookname_to_ifo.at(line));
+                auto it = bookname_to_ifo.find(line);
+                if (it != bookname_to_ifo.end()) {
+                    order_list.push_back(it->second);
+                }
```

Comment:
If there is error in file with list of dictionaries, may be better report that to user?